### PR TITLE
Add update for effects + colors and resources for effect parameters

### PR DESCRIPTION
**Pull request by 4ian**
This adds support for colors, resources in properties and an update method called at every frame for effects.

### CODE REVIEW

**Comment by blurymind** [comment]
Thank you @4ian :) I need to update my pr

**Comment by 4ian** [comment]
@blurymind I commented about the changes here: https://github.com/4ian/GDevelop/pull/1325#issuecomment-568173291 :) 

Be sure to re-run npm start to get the latest version of libGD.js that was just finished building moments ago: https://travis-ci.org/4ian/GDevelop/builds/628066457

File: Extensions/ExampleJsExtension/JsExtension.js [hunk 1]
```diff
@@ -33,6 +33,27 @@ module.exports = {
...
+        .setLabel(
+          _("Image resource (won't be used, just for demonstration purpose)")
+        )
+    );
+    dumyEffectProperties.set(
+      'someColor',
+      new gd.PropertyDescriptor(/* defaultValue= */ "#0022FF")
+        .setLabel(_('Some color for the effect'))
+        .setType('color')
+        .setLabel(
```
Comment (Bouh) [inline Extensions/ExampleJsExtension/JsExtension.js]: There are two setLabel()

Comment (4ian) [reply Extensions/ExampleJsExtension/JsExtension.js]: Good catch ;) I've fixed this in master.

**Pull request merged**